community/cypher/docs/cypher-docs/src/docs/dev/ql/using/index.asciidoc

```diff
@@ -2,10 +2,11 @@
 = Using
 
 [abstract]
-`USING` is used to give Neo4j hints about which index to use, or to scan instead of using an index.
+`USING` is used to influence the decisions of the planner when building an execution plan for a query.
+NOTE: Forcing the planner can lead to badly performing queries, and should only be used as a last ditch effort by experienced developers and database administrators.
```

Comment:
I prefer 'Forcing planner behaviour is an advanced feature, and should be used with caution by experienced developers and/or database administrators only, as it may cause queries to perform poorly'.